jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/main/java/org/jbpm/console/ng/ht/client/editors/taskcomments/TaskCommentsViewImpl.java

```diff
@@ -48,12 +49,18 @@
 import org.jboss.errai.ui.shared.api.annotations.Templated;
 import org.jbpm.console.ng.ht.client.i18n.Constants;
 import org.jbpm.console.ng.ht.model.CommentSummary;
+import org.uberfire.ext.services.shared.preferences.GridGlobalPreferences;
+import org.uberfire.ext.services.shared.preferences.GridPreferencesStore;
 import org.uberfire.ext.widgets.common.client.tables.PagedTable;
 import org.uberfire.workbench.events.NotificationEvent;
 
 @Dependent
 @Templated(value = "TaskCommentsViewImpl.html")
 public class TaskCommentsViewImpl extends Composite implements TaskCommentsPresenter.TaskCommentsView {
+    public static final String COL_ADDEDBY="addedBy";
+    public static final String COL_ADDEDAT="addedAt";
+    public static final String COL_COMMENT="comment";
+    public static final String COL_ID_ACTIONS = "Actions";
```

Comment:
please review code formatting. Perhaps adding a line break between the list of variables and the class declaration would also be nice. Consider using protected visibility so it is restricted to manly this class + test.